src/utils.ts

```diff
@@ -11,3 +11,25 @@ export const fontSize: (theme: TockTheme) => number = (theme: TockTheme) => {
   }
   return size;
 };
+
+/**
+ * Retrieves and returns an object from the local storage if found.
+ * If the value is not found it will save an initialValue before returning it.
+ * @param key - key in local storage
+ * @param computeInitialValue - function to create an initial value if the object is not found
+ */
+export const fromLocalStorage = (key: string, computeInitialValue: () => any) => {
+  try {
+    const item = window.localStorage.getItem(key);
+    if (item) {
+      return JSON.parse(item);
+    } else {
+      let initialValue = computeInitialValue();
```

Comment:
You can use `const` instead of `let`, it indicate other devs that the value is not changing.

Reply:
ok 👍